dashboard/v1.1/src/pages/Monitoring/Matrix.tsx

```diff
@@ -38,6 +38,17 @@ const round = (n: string): number => {
   return Math.round(num * 10) / 10;
 };
 
+const formatResLength = (resLength: number): string => {
+  if (resLength > 9999999999) {
+    return (resLength / 1000000000).toFixed(2).toString() + ' B';
+  } else if (resLength > 999999) {
+    return (resLength / 1000000).toFixed(2).toString() + ' M';
+  } else if (resLength > 999) {
+    return (resLength / 1000).toFixed(2).toString() + ' K';
+  }
+  return resLength.toString();
+};
+
```

Comment:
can we have this from `utils`?

Also, not sure if there should be a space between B (units) and the value. Please see if it looks good with or without the space.

Also, can we use `switch-case` instead of a `if-else-if` block?

Reply:
@Harkishen-Singh I have moved the functions to the utils folder. But for using `switch-case`, I think as the comparison is over a  range of values so it will be good to use `if-else-if`.